src/style.css

```diff
@@ -1 +1,115 @@
-/* add styles here */
+.html {
+  margin: 0;
+  padding: 0;
+}
+
+@font-face {
+  font-family: Avenir;
+  src:
+    url(./fonts/Avenir-Book.ttf) format('Avenir'),
+    url(./fonts/Avenir-Heavy.ttf) format('Avenir'),
+    url(./fonts/Avenir.ttc);
+}
+
+input {
+  border: none;
+  outline: 0;
+}
+
+input::placeholder {
+  font-family: Avenir, sans-serif;
+  font-weight: 300;
+  color: #3d4e61;
+}
+
+.wrap {
+  display: flex;
+  flex-direction: column;
+  align-items: center;
+  justify-content: space-between;
+}
+
+.form {
+  position: relative;
+  width: 100%;
+  background-color: rgba(255, 255, 255, 1);
+  border: 1px solid rgba(225, 231, 237, 1);
+  box-sizing: border-box;
+  box-shadow: 0 1px 8px 0 rgba(61, 78, 97, 0.1);
+  border-radius: 4px;
+}
+
+.big--form {
+  margin-top: 12px;
+  height: 70px;
+}
```

Comment:
It's generally not a good practice to set fixed heights on elements. Content can vary and may not fit within the fixed height, leading to overflow issues.